test/Tags/Spec.hs

```diff
@@ -102,5 +88,5 @@ spec = do
         , Tag "foo" Method (Loc (Range 166 184) (Span (Pos 18 3) (Pos 19 6))) "def self.foo" Nothing
         ]
 
-symbolsToSummarize :: [Text]
-symbolsToSummarize = ["Function", "Method", "Class", "Module"]
+parseTestFile :: Foldable t => t Tags.Kind -> Path.RelFile -> IO [Tag]
+parseTestFile include path = runTaskOrDie $ readBlob (fileForPath (Path.toString path)) >>= runReader defaultLanguageModes . fmap (filter ((`elem` include) . kind)) . tagsForBlob
```

Comment:
Abstracting over the parser has been accomplished by the use of `tagsForBlob` (and for convenience, a filter to eliminate the kinds we don’t care about).